ubuntudesign/vanilla-framework
# Audit links

## Done

- Updated base styling of links
- Update `.external` link pattern to become `.p-link--external`
- Update `.link-top` link pattern to become `.p-link--top`
- Update documentation
- Temporarily add back `demo` folder for QA purposes

## QA

- Sanity check code
- Run `gulp develop` and check `demo/index.html` for new and improved links as above.

## Details

Fixes: #417
